bpmn-i18n.xsd

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<schema
-  xmlns="http://www.w3.org/2001/XMLSchema"
+<xsd:schema
+  xmlns="http://www.omg.org/spec/BPMN/non-normative/extensions/i18n/1.0"
```

Comment:
I'm not sure I agree with this because this an XML Schema file and not an instance of "BPMN I18n". Setting `xmlns` to the i18n namespace, forces prefixing everything in the document with the `xsd:` namespace prefix. But that makes the schema harder to read and maintain.

Are you trying to fix a particular problem with this schema?

Reply:
Yes. The whole purpose of this PR is to make it possible to consume the XSD (fix it so it works). This particular fix ensures that `tTransaction` is declared in the correct namespace, cf. https://github.com/bpmn-miwg/bpmn-i18n/pull/7/commits/25b2c2ad3860672603c3911fb217a11f8bdb473d.

Reply:
With our tooling we rely on a working XSD schema, as we validate diagrams during our roundtrip tests, cf. https://github.com/bpmn-io/bpmn-i18n-moddle/pull/2.

Reply:
> I'm not sure I agree with this because this an XML Schema file and not an instance of "BPMN I18n". 

I do believe there is a value in being consistent with existing BPMN related standards (and their schemata), too. In particular consistently prefixing with `xsd` and using the namespace in the way I proposed matches 100% with the way [the BPMN 2.0 schema is defined](https://github.com/bpmn-io/bpmn-moddle/blob/master/resources/bpmn/xsd/Semantic.xsd#L2).

Reply:
Sorry, never mind. It looks like I still don't understand XML namespaces after 15 years ;-P

Reply:
No worries. I had to write two XML parsers (including namespace parsing logic) and I still need to integration test against a real parser every time.